packages/components/src/layout/tests/jest/Footer.test.tsx

```diff
@@ -0,0 +1,56 @@
+import { render, screen } from "@hopper-ui/test-utils";
+import { createRef } from "react";
+
+import { Footer } from "../../src/Footer.tsx";
+import { FooterContext } from "../../src/FooterContext.ts";
+
+describe("Footer", () => {
+    it("should render with default class", () => {
+        render(<Footer data-testid="Footer">Test</Footer>);
+
+        const element = screen.getByTestId("Footer");
```

Comment:
i try to stick to this order of priority for Testing Library, do you think that make sense in this case to use getByText?
https://testing-library.com/docs/queries/about/#priority